### ISSUE TITLE
seq2seq 챗봇관련 질문이 있습니다. 

### ISSUE THREAD

**Issue by taeheonkwon** [NLP-kr/tensorflow-ml-nlp-tf2#39]
안녕하세요...

"텐서플로 2와 머신러닝으로 시작하는 자연어처리" 책을 보다가 이렇게 질문을 드리게 되었습니다. 양해 바랍니다..

seq2seq 로 만든 챗봇 예제(306 ~ 317 page) 관련 2가지 질문이 있습니다. 

1. 예제에서는 디코더 클래스의  call 함수가 반환하는 x 값이 predictions 값으로 되어 있습니다. 그런데 x 값은  self.fc = tf.keras.layers.Dense(self.vocab_size) 에서 활성화함수 없이 그대로 출력되고 있습니다. softmax 함수를 거치지 않고 그냥 출력값이 높은 단어 1개를 정답으로 선택하는 건가요? 부연 설명을 부탁 드립니다....

질문을 많이 드려 죄송합니다.. 제가 부족한 점이 많습니다....

고맙습니다...
